src/pytest_databases/docker/elastic_search.py

```diff
@@ -0,0 +1,127 @@
+# MIT License
```

Comment:
why do we do this on each file?

Reply:
I've enabled the License required check for ruff.  I have this requirement internally, and I find that it's helpful to be consistent. 

If this is not something we want to do to our repos, then I can remove it.